tests/integration/test_install_twists.py

```diff
@@ -8,6 +8,7 @@
 import pytest
 
 from flaky import flaky
+import delegator
```

Comment:
I guess this line can be removed?

Reply:
Ah, good point. That was left over from an earlier iteration. I'll fix that.